docs/common-concepts/event-bus.mdx

```diff
@@ -0,0 +1,185 @@
+article that describes a concept where the concept's name can be the
+article's title (e.g., "Vert. x", "Verticle", "Event Bus")
+
+=== When not to use this template:
+Do not use these templates for articles that give context on how
+to use/approach the topic. Instead, it would be best if you used a guide for that.
+
+Writing a guide and a concept article are not mutually exclusive. Instead,
+you should, in most cases, write both, where the concept gives a brief overview
+of what it is, and the guide is a longer article explaining best practices on using
+it.
+
+E.g.:
+Concept: Event Bus
+Guide: Interacting with the Event Bus
+
+
+=== Writing tips:
+- Write in the present tense
+- Use neutral pronouns (they/them instead of he/she and him/her)
+- Be respectful to everyone
+- Be aware of the potential for cultural misunderstandings
+-->
+
+<!-- Relevant imports: -->
+
+import { Reference, Image } from '/components';
+
+<!-- Brief Description (1-3 lines) of the concept: -->
+
+The Event Bus is, basically, the nervous system of the Telestion Ecosystem. The
+ecosystem uses the Event Bus for the communication between its various
+components.
+
+## Video version
+
+You can find a video version of this concept's explanation here:
+
+<Reference to="https://www.youtube.com/watch?v=X92A77OdEg0">
+	Event Bus | Telestion Concepts (Video)
+</Reference>
+
+## Messages
+
+At the core of the Event Bus, there are so-called _Messages_. Messages are data
+packets that you can send to a specific channel. Then, all components monitoring
+that channel receive you message.
+
+## Channels and Addresses
+
+A _Channel_ is defined by its _Address_ (or _Channel Address_), which is a
+string (text) and essentially serves as the channel's name.
+
+<Image
+	invertible
+	src="/static/img/drawio-diagrams/vertx/eventbus.drawio.svg"
+	center
+/>
+
+The address typically describes the channel's content (e.g., `raw-telemetry`,
+`telecommand-out`, etc.).
+
+:::tip Mental model
+
+As a mental model, you can compare the event bus to radio communication, e.g.,
+in Air Traffic Control:
+
+| Concept in Radio Communication | Corresponding concept in the Event Bus |
+| ------------------------------ | -------------------------------------- |
+| Radio Transmission             | Message                                |
+| Frequency                      | Address                                |
+
+:::
+
+## Data Pipeline
+
+Through the combination of more than one component
+([Verticles](/application/concepts/verticle/) on the _Application_ side) sending
+each other messages and performing small, atomic tasks, you can create a sort of
+data pipeline that's both easier to think about (as individual components only
+need to perform smaller tasks) and more adjustable (since you can just plug
+components together in different ways):
+
+<Image
+	invertible
+	src="/static/img/drawio-diagrams/vertx/data-pipeline.drawio.svg"
+	center
+/>
+
+## Requests
```

Comment:
If you describe the point-to-point to communication in Vert.x, I think it's useful to describe the differences between Broadcast (Publish and Subscribe) and Unicast (Request and Reply) communication in Vert.x, too.

Maybe move the event bus image from above in a new section named "Broadcast Communcation" and rename this section to "Unicast Communication"? :thinking:

Reply:
I don't think that we need the full explanation of this, here. It would add a complexity that's not required for working with the Event Bus (and thus, isn't necessary in an introductory article).

But I agree that we need to explain the handling (and how it's different from the model described in the section above). Thus, I think I might add an info callout/admonition about "Requests get handled by one of the registered handlers, chosen at random".